command/CommandController.js

```diff
@@ -201,6 +216,20 @@ async function processArticleRequest(req, summaryOnly, extendedData) {
   } else {
     result = { url: audioUrl };
   }
+
+  if (endInstructions) {
+    let expireDate = new Date();
+    // Set the expire_date to 30 days ago
+    expireDate.setDate(expireDate.getDate() - 30);
+    if (new Date(endInstructionsData.date) < expireDate) {
+      endInstructionsData.url = await buildAudioFromText(
+        endInstructionsData.text
+      );
+      endInstructionsData.date = Date.now();
+    }
+    result.instructions_url = endInstructionsData.url;
```

Comment:
So we are just returning the synthesized instructions to the client and not synthesizing the entire file together?  Doesn't that make more work for the skill?

Reply:
I just realized that we can't reuse the same file for the mobile then.  I guess this makes the most sense at this point to return it as a separate url.

Reply:
Yeah we need to have separate files, as we store the article for both mobile and Alexa. I made another PR with the Alexa skill changes to enqueue the instructions at the end.

At the moment, we have the same instructions prompt for every article, so it is also more memory-friendly and cost-friendly as we synthesize less text than if we add the instructions to each article.